### PR TITLE
Adding section motivations

### LINKED ISSUE #104: Each step should carry an intro text
Each step should carry a short intro of what the step is about, why that step is important and what will a student achieve by the end of the step

### PR DESCRIPTION
**Pull request by yevster**
Fixes #104

### CODE REVIEW

File: 00-setup-your-environment/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 __This guide is part of the [Azure Spring Cloud training](../README.md)__
 
-Setting up all the necessary prerequisites in order to expeditiously complete the lab.
+In this section, we'll set up everything you need to expeditiously complete the tutorial.
```
Comment (mnriem) [inline 00-setup-your-environment/README.md:5]: Replace tutorial with training ;)

File: 03-configure-monitoring/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 __This guide is part of the [Azure Spring Cloud training](../README.md)__
 
-Configure aggregation of Spring Boot applications logs to understand common issues.
+Now that we have an application deployed, we'll configure Log Analytics, so that we can quickly search the application's logs if something goes wrong. We'll see how to take advantage of Log Analytics in a later section, but because it takes time for log entries to start coming in, we'll do the configuration steps here before moving on with the tutorial.
```
Comment (mnriem) [inline 03-configure-monitoring/README.md:5]: Replace tutorial with training

File: 06-build-a-reactive-spring-boot-microservice-using-cosmosdb/README.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
 
 __This guide is part of the [Azure Spring Cloud training](../README.md)__
 
-Build a reactive Spring Boot microservice that uses the [Spring reactive stack](https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html) and is bound to a [Cosmos DB database](https://docs.microsoft.com/en-us/azure/cosmos-db/?WT.mc_id=azurespringcloud-github-judubois) in order to access a globally-distributed database with optimum performance.
+In this section, we'll build an application that uses a [Cosmos DB database](https://docs.microsoft.com/en-us/azure/cosmos-db/?WT.mc_id=azurespringcloud-github-judubois) in order to access a globally-distributed database with optimum performance.
+
+We'll use the reactive programming paradigm to build our microservice in this section, leveraging the [Spring reactive stack](https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html). In contrast, we'll build a more traditional data-driven microservice in the next section.
```
Comment (mnriem) [inline 06-build-a-reactive-spring-boot-microservice-using-cosmosdb/README.md:7]: And we'll ....

And I would remove the "In contrast" sentence here and mention something in the section where it is actually build

File: 07-build-a-spring-boot-microservice-using-mysql/README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 __This guide is part of the [Azure Spring Cloud training](../README.md)__
 
-Build a classical Spring Boot application that uses JPA to access a [MySQL database managed by Azure](https://docs.microsoft.com/en-us/azure/mysql/?WT.mc_id=azurespringcloud-github-judubois).
+In this section, we'll build another data-driven microservice. This time, we will use a classic relational database, a [MySQL database managed by Azure](https://docs.microsoft.com/en-us/azure/mysql/?WT.mc_id=azurespringcloud-github-judubois). And we'll use Java Persistence API (JPA) to access the data in a way that will be more familiar to non-Spring Java developers.
```
Comment (mnriem) [inline 07-build-a-spring-boot-microservice-using-mysql/README.md:5]: Remove the non-Spring Java developers here as both Spring and non-Spring developers use JPA

File: 06-build-a-reactive-spring-boot-microservice-using-cosmosdb/README.md (hunk 3 above)
Comment (yevster) [reply 06-build-a-reactive-spring-boot-microservice-using-cosmosdb/README.md:7]: Chewed on this... have to disagree. Saying "in contrast" creates a relationship between the two things. I think it makes it clearer that what we're showing is a "novel" paradigm, as opposed to a traditional one coming up.

**Pull request merged**